app/js/controllers/events.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular
+.module('app.controllers')
+.controller('EventsCtrl', function($scope, $location, Restangular) {
+	$scope.loadingPromise = Restangular.one('events/next-10-publicly')
```

Comment:
100% agree. Back story: we went with the `/next-10-publicly` style route because my lib didn't (doesnt) support pagination, and I thought it would be confusing for a user that they could request `/upcoming? page=whatever` (which would be easy to support) but then not put a pagination parameter on other routes, like `/events`. However, this route has been bothering me a lot, so I already [changed it](https://github.com/TechAtNYU/api/commit/9446b37e4245c28980d208c1095e2ab6df23783f) to support pagination parameters. That will introduce the aforementioned inconsistency, but at least that's a consistency we can fix over time in a forward compatible way (ie. by supporting the same parameters on other routes)